apps/docs/src/guide/wallets/test-wallets.md

```diff
@@ -1,13 +0,0 @@
-# Setting up test wallets
-
-You'll often want to create one or more test wallets when testing your contracts. Here's how to do it.
-
-## Create a single wallet
-
-<<< @/../../docs-snippets/src/guide/wallets/access.test.ts#wallets{ts:line-numbers}
-
-## Setting up multiple test wallets
-
-If you need multiple test wallets, they can be set up as follows:
-
-<<< @/../../docs-snippets/src/guide/wallets/test-wallets.test.ts#wallet-setup{ts:line-numbers}
```

Comment:
What uses do you see for it that won't be superseded by `WalletConfig`? Note that ideally we want other users to start using `launchTestNode` exclusively for tests and not give mixed messages by providing two approaches.

Reply:
That would be done the following way:
```ts
using launched = await launchTestNode();
const {wallets: [genesisWallet], provider} = launched;
const wallet = WalletUnlocked.generate({provider});

// do tests on non-funded wallet

await genesisWallet.fund(wallet.address, 1234);

// do tests on funded wallet
```

`generateTestWallet` serves the purpose of having an unique, funded wallet you can work with in your tests and which doesn't experience the "UTXO already spent" issues because of other tests using it simultaneously. With `WalletConfig` you get the same thing.

Its implementation is also tied to the specific way we've set up our current test suite and guided our users to do as well (e.g. dependency on `process.env.GENESIS_SECRET`, sharing the same node between tests, etc.). That approach will be deprecated once this PR gets merged and that's the reason why I'm removing it from the docs.

Reply:
> Just a thought - is it worth converting the following [test](https://github.com/FuelLabs/fuels-ts/blob/ns/feat/test-node-launcher/apps/docs-snippets/src/guide/wallets/test-wallets.test.ts) to using launchTestNode, rather than removing it entirely?

Not really, because `WalletConfig` examples cover it and there's no practical differentiation between an address that's been there since the genesis block (added via `WalletConfig`) and one that was created later (funded via `generateTestWallet`) which would necessitate translation of that linked example.